src/poliastro/constants/rotational_elements.py

```diff
@@ -0,0 +1,209 @@
+from astropy.constants import Constant
+
+T_earth = Constant(
```

Comment:
Could you please point me out to where exactly are these constants defined in Vallado? In the dictionary of symbols, `T` is not used for rotational period.

Reply:
It's not used so I should've probably replace it with ``rotational_period_`` (that is if we decide to keep this constant at all)